CHANGES_NEXT_RELEASE

```diff
@@ -2,4 +2,6 @@
 [cygnus-ngsi][CKANSink] New datamodel for CKAN (dm-by-entity-id) implementing mapping: subservice -> org, entityId -> dataset, entityId -> resource (#1792)
 [cygnus-ngsi][PostgisSQLSink, PostgreSQLSink] Add dm-by-entity-database and dm-by-entity-database-schema.
 [cygnus-ngsi][Generic Aggregation] If name mappings is enabled, then the aggregation will take all attributes from the mapped element on the event.
+[cygnus-common][pom.xml] Update Hive-jdbc dependency from version 2.3.4 to version 3.1.2
```

Comment:
Checking this with @IvanHdzC and @AlvaroVega , we wonder why this upgrade is needed. I mean, it is just a routinary package version upgrade? Or is there any hive-jdbc functionality/fix need by cygnus-ngsi-ld that exists in 3.1.2 but not in 2.3.4?

Reply:
Hello, any update on this?

Reply:
We are still waiting for the answers to these questions:

* Is it just a routinary package version upgrade? 
* Or is there any hive-jdbc functionality/fix need by cygnus-ngsi-ld that exists in 3.1.2 but not in 2.3.4?

Thanks!

Reply:
(In addition, an upgrade to CHANGES_NEXT_RELEASE in master due to recent 2.2.0 release yesterday has caused a minor conflict in this PR. Could you solve it so the PR gets mergeable again, please? Thanks!)

Reply:
Done 917bf6a

Reply:
What about the pending questions @anmunoz ? Any update on them, pls? Thx!

Reply:
Hello, is just a routinary update, when we compile cygnus from sources  we get an error about this dependency and the code for hive Sink, that is the reason why we decided to update it. Cygnus-ngsi-ld currently don't work with hive so that is not a necessary dependency for this development.

Reply:
Thanks for the answer!

Taking into account that change is not related to the purpose of the PR itself (cygnus-ngsi-ld) and that we (@AlvaroVega @IvanHdzC @fgalan ) have some doubts about compatibility (we haven't been able to trace the 2.3.4 to 3.1.2 changelog in that dependency) please move that change in pom.xml (and associated CHANGES_NEXT_RELEASE entry) to a separate PR so we can continue on it and this PR (the one in cygnus-ld) can be merged now.

Reply:
done in 1dc0f55 and  75c3a8d